.github/workflows/build.yaml

```diff
@@ -30,7 +30,8 @@ jobs:
 
         strategy:
             matrix:
-                type: [gcc_debug, gcc_release, clang, mbedtls]
+                type: [gcc_debug, gcc_release, clang, mbedtls, python_lib]
```

Comment:
we switched off matrix - this will require a conflict resolution.

Reply:
Just noticed #9023 was not yet merged, however I expect it will be soon. I would prefer to switch out of using matrix builds.

Reply:
Ok. Updated.